src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/HmppsHistoricalPrisonerApi.kt

```diff
@@ -2,8 +2,11 @@ package uk.gov.justice.digital.hmpps.historicalprisonerapi
 
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.data.web.config.EnableSpringDataWebSupport
+import org.springframework.data.web.config.EnableSpringDataWebSupport.PageSerializationMode.VIA_DTO
 
 @SpringBootApplication
+@EnableSpringDataWebSupport(pageSerializationMode = VIA_DTO)
```

Comment:
was getting warnings about PageImpl so fix is to add this in - see https://docs.spring.io/spring-data/commons/reference/repositories/core-extensions.html#core.web.page.config